### DISCUSSION TITLE
USB 3.0 purpose in v3.0, and real time

### DISCUSSION THREAD

**Discussion by Data2Logic** [tomojitakasu/PocketSDR#30] [Q&A] [answered]
Is there any reason behind the use of USB3.0 in V3.0 ? 480 Mbps of USB 2.0 seem to be enough for 48Msp. I am not sure behind the mathematic of this

**Comment by tomojitakasu** [comment, marked as answer]
The data rate for FE 4CH needs:

2 (bits/sample) x 2 (IQ) x 4 (ch) x 48 (Msps) / 8 = 96 MB/s

USB 2.0 (480 Mbps) is not enough for the data rate.
